packages/edit-post/src/components/layout/index.js

```diff
@@ -123,6 +127,8 @@ function Layout( { styles } ) {
 			showBlockBreadcrumbs: select( editPostStore ).isFeatureActive(
 				'showBlockBreadcrumbs'
 			),
+			// translators: Default label for the Document in the Block Breadcrumb.
+			documentLabel: postTypeLabel || _x( 'Document', 'noun' ),
```

Comment:
Actually you'll have to remove the translators comment and add it as a second argument. Example here: https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/unit-control/utils.js#L126

Reply:
Is that right?

I always thought context was rather for disambiguation between strings. See [docs](https://codex.wordpress.org/I18n_for_WordPress_Developers#Disambiguation_by_context)

Translators comments provide an explanation for the string.

Reply:
Hm.. after reading more about it, it seems it can probably work better in this case. Thank you for pointing it out!